app/src/main/java/jp/co/cyberagent/stf/Service.java

```diff
@@ -92,8 +108,15 @@ public void onCreate() {
 
         clipboardManager = getSystemService(Context.CLIPBOARD_SERVICE);
 
+        String channelId;
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            channelId = createNotificationChannel("stf_channel", "STF Channel");
+        } else {
+           channelId="";
```

Comment:
Could you reformat the code to use consistent spacing, wrapping etc?

Reply:
Sure 👍 Done